lib/optional_query_cache/version.rb

```diff
@@ -0,0 +1,11 @@
+require 'active_record'
```

Comment:
You're right: it shouldn't live here, but making `OptionalQueryCache` a module with a `Middleware` class inside it seems… clunky to me?

Basically, what's happening is the gemspec tries to require this at `bundle install` time (when running on CI) and can't because it hasn't run `bundle install` yet… Catch 22 :boom: